commit ea9db201e65e324257c0e0797821794b8c105ed8
Author: Scott Moser <smoser@brickies.net>
Date:   Thu Aug 7 14:54:38 2025 -0400

    Add modules for cdrom probing.
    
    sr_mod is for generic scsi cdrom.
    ahci is to support qemu's -cdrom device (ide).

diff --git a/tools/ds-identify b/tools/ds-identify
index 99b91cbf0..99976f912 100755
--- a/tools/ds-identify
+++ b/tools/ds-identify
@@ -143,7 +143,7 @@ DI_EC2_STRICT_ID_DEFAULT="true"
 STATE_CDROM_PROBED=""
 STATE_FLOPPY_PROBED=""
 
-PROBE_CDROM_MODULES=""
+PROBE_CDROM_MODULES="ahci ata_piix sr_mod"
 _IS_IBM_CLOUD=""
 
 error() {
